DataFormats/L1THGCal/interface/HGCalMulticluster.h

```diff
@@ -23,7 +23,21 @@ namespace l1t {
       
       ~HGCalMulticluster() override;
 
+      bool hOverE() const { 
```

Comment:
realized now that in cleaning up the code I ended up with `bool` here instead of the `float` that should have been.
the good news is that just fixing the return type of the accessor is enough to get the right output since the variable was correctly stored as float.
I can make bugfixes later today, just let me know for what branches